blockchain/blockchain.go

```diff
@@ -454,6 +454,36 @@ func (bc *BlockChain) SetProposerPolicy(val uint64) {
 	bc.chainConfig.Istanbul.ProposerPolicy = val
 }
 
+func (bc *BlockChain) SetLowerBoundBaseFee(val uint64) {
+	bc.chainConfigMu.RLock()
```

Comment:
@kjeom Isn't RLock() for reader lock? It seems you update the value, so you need Lock() instead of RLock().

Reply:
Thanks, It should have Lock to write